constants/hardware_constants.asm

```diff
@@ -1,6 +1,6 @@
-; From http://nocash.emubase.de/pandocs.htm.
+; From https://problemkaputt.de/pandocs.htm
```

Comment:
Might as well link to the new and improved https://gbdev.io/pandocs/ imo.

Otherwise seems good to me.